### PR TITLE
All Comments Converted to JavaDoc Format

### PR DESCRIPTION
**Pull request by scarroll142**
Read the title.

### CODE REVIEW

File: umlobject/Segment.java [hunk 1]
```diff
@@ -82,10 +85,11 @@ public Segment(UMLNode inStart, UMLNode inStop) {
    }
 
   /**
-   * Standard Constructor
+   * Standard Constructor.
+   *
    * @param inStart starting UMLNode for line to be drawn between.
    * @param inStop stopping UMLNode for line to be drawn between.
-   * @param dotted flag determing whether Segment will contain a dotted line or not.
+   * @param isDotted the is dotted
```
Comment (tommy-russoniello) [inline umlobject/Segment.java]: ?

File: umlaction/UMLAction.java [hunk 2]
```diff
@@ -14,15 +13,54 @@
...
 public class UMLAction {
 
+  /**
+   * The Enum RelationshipType.
+   */
   public enum RelationshipType {
-		ASSOCIATION, DEPENDENCY, AGGREGATION, COMPOSITION, GENERALIZATION,
+		
+		/** 
+		 * The association realationship type.
```
Comment (tommy-russoniello) [inline umlaction/UMLAction.java]: relationship*

File: umlaction/MoveUMLNode.java [hunk 3]
```diff
@@ -3,24 +3,77 @@
...
   }
 
+  /**
+   * Instantiates a new move UML node.
+   *
+   * @param inNode the in node
+   * @param inOriginalX the in original X
+   * @param inOriginalY the in original Y
+   * @param inUMLNodeController the in UML node controller
+   * @param inMoveAtFirst the in move at first
```
Comment (tommy-russoniello) [inline umlaction/MoveUMLNode.java]: ???

File: umlaction/MoveUMLNode.java [hunk 4]
```diff
@@ -3,24 +3,77 @@
...
+ */
 public class MoveUMLNode extends UMLNodeAction {
+  
+  /** 
+   * The original x and y coordinates followed by the new x and y coordinates.
+   */
   double originalX, originalY, movedX, movedY;
+  
+  /** 
+   * Boolean representing whether move at first is true or false.
```
Comment (tommy-russoniello) [inline umlaction/MoveUMLNode.java]: ?

File: umlaction/ChangeRelationshipType.java [hunk 5]
```diff
@@ -6,13 +6,49 @@
...
+	 * The new connector. 
+	 */
 	UMLObject newConnector;
 
+	/**
+	 * Instantiates a new change relationship type.
+	 *
+	 * @param original the original
+	 * @param inNewRelationship the in new relationship
+	 * @param c the c
```
Comment (tommy-russoniello) [inline umlaction/ChangeRelationshipType.java]: ??

File: umlaction/AddUMLNode.java [hunk 6]
```diff
@@ -17,6 +17,13 @@
  * @since 0.3
  */
 public class AddUMLNode extends UMLNodeAction {
+  
+  /**
+   * Instantiates a new adds the UML node.
```
Comment (tommy-russoniello) [inline umlaction/AddUMLNode.java]: ?

File: juml/UMLNodeController.java [hunk 7]
```diff
@@ -1,13 +1,29 @@
 package juml;
 
+/**
+ * The Class UMLNodeController.
```
Comment (tommy-russoniello) [inline juml/UMLNodeController.java]: this should be the javadoc file header syntax like in the rest of the files

File: juml/Tests.java [hunk 8]
```diff
@@ -11,13 +11,30 @@
...
+	/** The juml. */
 	Controller juml;
+	
+	/** The p 3. */
 	Point p1, p2, p3;
+	
+	/** The cb 3. */
 	ClassBox cb1, cb2, cb3;
+	
+	/** The c 6. */
```
Comment (tommy-russoniello) [inline juml/Tests.java]: wat

File: juml/ObjectController.java [hunk 9]
```diff
@@ -1,5 +1,13 @@
 package juml;
 
+
+/**
+ * The Class ObjectController. Extended by UMLNodeContoller and UMLConnectorController extend.
```
Comment (tommy-russoniello) [inline juml/ObjectController.java]: should be javadoc file header syntax like in other files

File: juml/Main.java [hunk 10]
```diff
@@ -2,12 +2,14 @@
...
 import javafx.application.Application;
 import javafx.fxml.FXMLLoader;
-import javafx.scene.image.Image;
 import javafx.scene.layout.BorderPane;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
+/**
+ * The main class that launches the application.
```
Comment (tommy-russoniello) [inline juml/Main.java]: should be javadoc file header syntax like in other files

File: juml/Main.java [hunk 11]
```diff
@@ -43,8 +54,8 @@ public void start(final Stage primaryStage) throws Exception{
       window.setTitle("Team Rocket UML Editor: " + fileName);
 
       Scene scene = new Scene(root, 1000, 500);
-      window.getIcons().add(new Image(Main.class.getResourceAsStream(
-        "/images/Team_Rocket_Logo.jpg")));
+      //window.getIcons().add(new Image(Main.class.getResourceAsStream(
+      //  "/images/Team_Rocket_Logo.jpg")));
```
Comment (tommy-russoniello) [inline juml/Main.java]: ?

**Review by Tmilly — APPROVED** [review]
Looks good, great job guys!

**Review by quinnminnich — APPROVED** [review]

**Review by tommy-russoniello — APPROVED** [review]

**Pull request merged**